fenix/app/src/main/java/org/mozilla/fenix/home/ToolbarView.kt

```diff
@@ -57,6 +60,14 @@ class ToolbarView(
     }
 
     private fun updateLayout(view: View) {
+        if (IncompleteRedesignToolbarFeature(context.settings()).isEnabled) {
+            binding.menuButton.visibility = GONE
+            binding.tabButton.visibility = GONE
+        } else {
+            binding.menuButton.visibility = VISIBLE
+            binding.tabButton.visibility = VISIBLE
+        }
```

Comment:
This could be simplified using 

`isVisible = IncompleteRedesignToolbarFeature(context.settings()).isEnabled`